terraform/environments/ci/main.tf

```diff
@@ -0,0 +1,22 @@
+terraform {
+    backend "s3" {
+        bucket = "hashbang-terraform-ci"
+        key = "state.tfstate"
+        region = "us-west-2"
+        lock_table = "hashbang-terraform-ci"
+    }
+}
+
+provider "aws" {
+    region = "us-west-2"
+}
+
+module "vpc" {
+    source = "github.com/terraform-community-modules/tf_aws_vpc?ref=v1.0.6"
+    name = "ci"
+    cidr = "10.0.0.0/16"
```

Comment:
Just giving a 10/16 to each enviornment of which will initally only be Production and CI. Keeping them far apart to have plenty of addresses for throw-away workers etc in the future.